js/views/itemEditVw.js

```diff
@@ -663,5 +663,10 @@ module.exports = baseVw.extend({
     this.validateDescription();
 
     return this.$('#contractForm')[0].checkValidity();
+  },
```

Comment:
Ah, ok. I noticed it wasn't there while trying to hunt down why the data wasn't getting cleared. I'll take it out.